runtime/foucoco/src/lib.rs

```diff
@@ -246,10 +246,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("amplitude"),
 	impl_name: create_runtime_str!("amplitude"),
```

Comment:
One last thing. Should we also take this opportunity to rename these to `foucoco`? It's a bit confusing that this is also called "amplitude". WDYT @TorstenStueber @adelarja ?

Reply:
Sounds great to me!